domains/tools.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "asma019",
+        "email": "asmaparvin019@gmail.com"
+    },
+    "record": {
+        "CNAME": "asma019.github.io"
```

Comment:
Little content provided on site.

Reply:
I want to create web base tool site with this domain.